src/test/java/com/leaf/billing/FileUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.leaf.billing;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoField;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class FileUtilTest {
+
+    @Test
+    void extractTimestampFromFileName() {
+        String name = "boundaryEvent2_2022-01-01T13-00-00.parquet";
+        LocalDate date = com.leaf.billing.utils.FileUtils.extractTimestampFromFileName(name);
+        assertThat(date.get(ChronoField.YEAR)).isEqualTo(2022);
+        assertThat(date.get(ChronoField.MONTH_OF_YEAR)).isEqualTo(1);
+        assertThat(date.get(ChronoField.DAY_OF_MONTH)).isEqualTo(1);
```

Comment:
```suggestion
    void extractTimestampFromFileName() {
        String name = "boundaryEvent2_2022-05-01T13-00-00.parquet";
        LocalDate date = com.leaf.billing.utils.FileUtils.extractTimestampFromFileName(name);
        assertThat(date.get(ChronoField.YEAR)).isEqualTo(2022);
        assertThat(date.get(ChronoField.MONTH_OF_YEAR)).isEqualTo(5);
        assertThat(date.get(ChronoField.DAY_OF_MONTH)).isEqualTo(1);
```

I know it probably works, but having the test assert the same value for different properties might not be ideal.